drivers/jenkins/src/main/kotlin/com/gatehill/corebot/driver/jenkins/action/JenkinsJobTriggerService.kt

```diff
@@ -231,4 +231,8 @@ class JenkinsJobTriggerService @Inject constructor(private val actionDriver: Jen
             }
         })
     }
+
+    override fun fetchExecutionOutput(trigger: TriggerContext, action: ActionConfig, executionId: Int) {
+        //TODO: implement.
```

Comment:
Could do with a warning log entry here to indicate it's not supported for this driver.